test/tests/biguser.test.ts

```diff
@@ -126,7 +125,8 @@ it('should create a user with [Plugin].findOrCreate', async () => {
   expect(foundUser.doc).toHaveProperty('firstName', 'Jane');
 
   try {
-    await UserModel.create({
+    // @ts-ignore
```

Comment:
why is an `ts-ignore` here?

Reply:
I think the `languages` key is required but it isn't specified here. Might want to change the schema instead of monkeypatching this call, though.